src/faction_camp.cpp

```diff
@@ -1698,7 +1702,10 @@ void basecamp::worker_assignment_ui()
 
         ui_manager::redraw();
         const std::string action = ctxt.handle_input();
-        if( action == "DOWN" ) {
+        std::string sInput = ctxt.get_raw_input().text;
+        if (sInput == "i") {
```

Comment:
This should be added as an action so player can change the keybindings. (Edit `data/raw/keybindings.json`)

Reply:
Done, thx